go-fuzz-build/main.go

```diff
@@ -71,10 +75,25 @@ func main() {
 	}
 
 	deps := make(map[string]bool)
+	mu := sync.Mutex{}
+	wg := sync.WaitGroup{}
+	sem := make(chan bool, Par)
 	for _, p := range goListList(pkg, "Deps") {
-		deps[p] = goListBool(p, "Standard")
-	}
+		wg.Add(1)
+		sem <- true
+		go func(p string) {
+			res := goListBool(p, "Standard")
```

Comment:
If `go list` invocation is the problem, then we should extract all interesting properties and once. Currently we run `go list` 4 times per-package: to check Standart, extract Import, GoFiles CgoFiles.